docs/model_repository.md

```diff
@@ -169,6 +171,68 @@ $ export AZURE_STORAGE_ACCOUNT="account_name"
 $ export AZURE_STORAGE_KEY=$(az storage account keys list -n $AZURE_STORAGE_ACCOUNT --query "[0].value")
 ```
 
+### Cloud Storage with Credential file (Beta)
+
+*This feature is currently in beta and may be subject to change.*
+
+To group the credentials into a single file for Triton, you may set the 
+`TRITON_CLOUD_CREDENTIAL_PATH` environment variable to a path pointing to a 
+JSON file of the following format, residing in the local file system.
+
+```
+export TRITON_CLOUD_CREDENTIAL_PATH="cloud_credential.json"
+```
```

Comment:
Do we want to expose a `tritonserver ... --credential-file cloud_credential.json` ? 

Maybe not include `cloud` in the name so this arg can be re-used for future work too?